source/reference/kubectl-minio-plugin/kubectl-minio-tenant-upgrade.rst

```diff
@@ -0,0 +1,88 @@
+
+.. _kubectl-minio-tenant-upgrade:
+
+================================
+``kubectl minio tenant upgrade``
+================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: kubectl minio tenant upgrade
+
+Description
+-----------
+
+.. include:: /includes/facts-kubectl-plugin.rst
+   :start-after: start-kubectl-minio-requires-operator-desc
+   :end-before: end-kubectl-minio-requires-operator-desc
+
+.. start-kubectl-minio-tenant-upgrade-desc
+
+Upgrades the ``minio`` server container image used by the MinIO Tenant.
+
+.. end-kubectl-minio-tenant-upgrade-desc
+
+.. important::
+
+   MinIO upgrades the image used by all pods in the Tenant at once. 
+   This may result in downtime until the upgrade process completes.
```

Comment:
```
   Applications typically transparently retry operations against the MinIO Tenant, such that there should be no perceived downtime.
```

I've been given guidance to avoid referring to downtime, as MinIO restarts fast enough where this isn't an issue.

Reply:
Maybe also add here "Test all upgrades in a staging environment, such as a separate MinIO Tenant, before applying to your production tenants"

Reply:
👍